libraries/opensk/src/api/persist.rs

```diff
@@ -262,17 +264,17 @@ pub trait Persist {
     /// available. This includes cases of offset being beyond the stored array.
     ///
     /// The buffer is passed in when writing is in process.
-    fn get_large_blob(
-        &self,
+    fn get_large_blob<'a>(
+        &'a self,
```

Comment:
This seems wrong to me. We don't want the result to borrow from self.

```suggestion
        &self,
```

Reply:
I blindly followed the compiler suggestion, but I agree it felt wrong. Yours is correct!